### PR TITLE
lib update & allo php 8

### PR DESCRIPTION
**Pull request by Kalipso0505**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "php": "^7.1"
+        "php": ">=7.1"
```
Comment (kenowessels) [inline composer.json]:
```suggestion
        "php": "^7.1 || ^8.0"
```

Comment (Dgame) [reply composer.json]: `"^7.1 || ^8.0"` afaik

**Comment by Dgame** [comment]
I would be happy if you also submit the changes to the original repo, so that others who might work with it can also benefit from your change. 🙂

**Comment by Kalipso0505** [comment]
> I would be happy if you also submit the changes to the original repo, so that others who might work with it can also benefit from your change. slightly_smiling_face

I didn't had the rights to push a branch on your repo. So, to be honest, i don't how to this via GitHub.

**Comment by Dgame** [comment]
After you've forked a project and created a PR, you can initially decide whether the base should be the original project or the forked project. I always prefer to set the base to the original. In this case, I can simply adjust the code myself, but I would recommend this approach for the future.

**Pull request merged**